assets/js/frontend/single-course.js

```diff
@@ -0,0 +1,20 @@
+jQuery( document ).ready( () => {
```

Comment:
I wonder if it makes more sense to put this with the block files. Maybe `assets/blocks/course-outline/view.js` or `frontend.js`?

Reply:
Done in 7e12159da1cec52bbe3ca118792bef16101690c1